static/templates/compose_select_enter_behaviour_popover.hbs

```diff
@@ -0,0 +1,36 @@
+<div class="enter_sends_choices grey-box">
+    <label class="enter_sends_choice">
+        <input type="radio" class="prop-element" value="true"{{#if enter_sends_true }} checked{{/if}} />
+        <div class="enter_sends_choice_text">
+            <span>
+                {{#tr}}
+                    <z-button><kbd>Enter</kbd></z-button>
+                    <span>to send</span>
+                {{/tr}}
+            </span>
+            <span class="enter_sends_minor">
+                {{#tr}}
+                    <z-button><kbd>Ctrl</kbd> + <kbd>Enter</kbd></z-button>
+                    <span>to add a new line</span>
+                {{/tr}}
+            </span>
+        </div>
+    </label>
+    <label class="enter_sends_choice">
+        <input type="radio" class="prop-element" value="false"{{#unless enter_sends_true }} checked{{/unless}} />
+        <div class="enter_sends_choice_text">
+            <span>
+                {{#tr}}
+                    <z-button><kbd>Ctrl</kbd> + <kbd>Enter</kbd></z-button>
+                    <span>to send</span>
+                {{/tr}}
+            </span>
+            <span class="enter_sends_minor">
+                {{#tr}}
+                    <z-button><kbd>Enter</kbd></z-button>
+                    <span>to add a new line</span>
+                {{/tr}}
+            </span>
```

Comment:
Is this what you meant?

Reply:
Do we need the span?  I was thinking something more like:
```
                {{#tr}}
                    <z-key>Enter</z-key> to add a new line
                {{/tr}}

```

with the `z-key` thing generating the `kdb` element as well.

Reply:
Well, we need to identify `kbd` elements differently so that they can be replaced by mac hotkeys on mac systems. So, we need atleast` <z-button><kbd>Ctrl</kbd>+<kbd>Enter</kbd></z-button>`. Having a `span` is good to apply different style to text than the `kbd` and is recommended.

Reply:
<img width="1508" alt="Screenshot 2021-12-17 at 10 36 56 PM" src="https://user-images.githubusercontent.com/25124304/146584829-94690399-0403-4adc-89ed-d962ff3bfd0d.png">
This is still not correct, though.

Reply:
Huh, I'd thought the the `kbd` replacement for Macs was done in JavaScript in a post-processing step after template rendering, and thus it doesn't matter how we represent things for translators as long as the final HTML is right?